src/bootstrap/index.ts

```diff
@@ -5,7 +5,11 @@ import { TypeDocAndTSOptions, Application, TSConfigReader, ProjectReflection } f
 import { matcher } from 'micromatch';
 import { resolve } from 'pathe';
 
-export function bootstrapProject(options: Partial<TypeDocAndTSOptions>, filteringGlob?: string): ProjectReflection {
+export function bootstrapProject(
```

Comment:
I would add a comment documenting the nodeModulesInputFilePaths and giving an example of input

Reply:
Added description to documentComponents function with an example